src/main/java/com/playtika/homework1/task4/FirstAndSecondPositions.java

```diff
@@ -0,0 +1,21 @@
+package com.playtika.homework1.task4;
+
+import java.util.Arrays;
+
+public class FirstAndSecondPositions {
+    public static void main(String[] args) {
+
+        String[] names = new String[]{"Elena", "Thomas", "Hamilton", "Suzie", "Phil", "Matt", "Alex", "Emma", "John", "James", "Jane", "Emily", "Daniel", "Neda", "Aaron", "Kate"};
```

Comment:
initialize array by this way `String[] names = {"Elena", ..}

Reply:
fixed